doc/cli.markdown

```diff
@@ -1212,18 +1212,25 @@ same as '--application'
 
 # Help and Version
 
-## help [command...]
+## help [command]
 
-Get detailed help for an specific command.
+List balena commands, or get detailed help for an specific command.
```

Comment:
`an specific` → `a specific` ? (I think.) (I know, it wasn't introduced by this PR.)

Reply:
Oops yes.